src/components/seo.js

```diff
@@ -24,35 +24,43 @@ function SEO({ description, lang, meta, keywords, title }) {
       }}
       meta={[
         {
-          name: `description`,
+          name: `description',
           content: metaDescription,
         },
+        {
+          property: 'og:title',
+          content: 'Mentor Night | Free Code Camp',
+        },
+        {
+          property: 'og:description',
+          content: 'A place to coach or be coached for free on your developer journey',
```

Comment:
I believe this should also reference `metaDescription`.

Reply:
Hey thanks, I'll run through these tomorrow!